great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -22,8 +25,15 @@
 try:
     import sqlalchemy as sa
     from sqlalchemy.engine import reflection
-    from sqlalchemy.sql.expression import BinaryExpression, literal
+    from sqlalchemy.sql.expression import (
+        BinaryExpression,
+        literal
+    )
     from sqlalchemy.sql.operators import custom_op
+    from sqlalchemy.sql.elements import WithinGroup
+    from sqlalchemy_redshift.dialect import RedshiftDialect
```

Comment:
If RedshiftDialect is *not* installed, including it together with base SqlAlchemy imports would cause fallback behavior inappropriately (and prevent import of PGDialect_psycopg2 and ProgrammingError)

It's not clear to me that the import is required at all based on the diff.